kornia/augmentation/container/ops.py

```diff
@@ -212,6 +222,9 @@ def transform(cls, input: Tensor, module: Module, param: ParamItem, extra_args:
                 "The support for 3d mask operations are not yet supported. You are welcome to file a PR in our repo."
             )
 
+        elif isinstance(module, K.MixAugmentationBaseV2):
```

Comment:
Probably like below? 
```suggestion
        elif isinstance(module, K.RandomTransplantation):
```